### ISSUE TITLE
Bug: Workspaces in Java 11 vs 8

### ISSUE THREAD

**Issue by Immurb** [jdemetra/jdemetra-app#439]
Workspaces created with JD+(Java 11) can't be opened with JD+(Java 8) and vice versa.

Error message:
```
java.io.IOException: Cannot probe workspace file format of 'WS_JavaX.xml'
	at ec.demetra.workspace.file.FileWorkspace.lambda$open$0(FileWorkspace.java:56)
	at java.base/java.util.Optional.orElseThrow(Optional.java:408)
	at ec.demetra.workspace.file.FileWorkspace.open(FileWorkspace.java:56)
[catch] at ec.nbdemetra.ws.FileRepository.load(FileRepository.java:194)...
```

The difference in the xml files seems to be the missing namespace in the Java 11 version. I attached my to test workspaces.
[WS_Java8.zip](https://github.com/jdemetra/jdemetra-app/files/2667783/WS_Java8.zip)
[WS_Java11.zip](https://github.com/jdemetra/jdemetra-app/files/2667784/WS_Java11.zip)

**Comment by charphi** [comment]
Thanks for finding this huge bug !

**Comment by charphi** [comment]
Same problem with Java 9.

**Comment by charphi** [comment]
https://bugs.java.com/bugdatabase/view_bug.do?bug_id=8132554
https://github.com/highsource/maven-jaxb2-plugin/issues/120

**Comment by charphi** [comment]
After some digging, I have found that the problem is not related to JAXB.
The problem is due to the failed loading of any `package-info.class` contained in a NetBeans module under Java9+.

NetBeans module provides its own class loader: `org.netbeans.SandardModule.OneModuleClassLoader`.
This class loader overrides some methods. Unfortunatly, Java9+ introduces a new method `ClassLoader#loadClass(Module, String)` that is not known by NetBeans module and is called in `Package#getPackageInfo()`. The result is that nothing is loaded at all.

```java
// jdk11
private Class<?> getPackageInfo() {
  if (packageInfo == null) {
    // find package-info.class defined by loader
    String cn = packageName() + ".package-info";
    Module module = module();
    PrivilegedAction<ClassLoader> pa = module::getClassLoader;
    ClassLoader loader = AccessController.doPrivileged(pa);
    Class<?> c;
    if (loader != null) {
      c = loader.loadClass(module, cn);
    } else {
      c = BootLoader.loadClass(module, cn);
    }

    if (c != null) {
      packageInfo = c;
    } else {
      // store a proxy for the package info that has no annotations
      class PackageInfoProxy {}
      packageInfo = PackageInfoProxy.class;
    }
  }
  return packageInfo;
}
  
// jdk8
private Class<?> getPackageInfo() {
  if (packageInfo == null) {
    try {
      packageInfo = Class.forName(pkgName + ".package-info", false, loader);
    } catch (ClassNotFoundException ex) {
      // store a proxy for the package info that has no annotations
      class PackageInfoProxy {}
      packageInfo = PackageInfoProxy.class;
    }
  }
  return packageInfo;
}
```

**Comment by charphi** [comment]
It is possible to force load `package-info` by using `Class.forName(String)`.

**Comment by charphi** [comment]
Fixed by b8ad2a2f3bbecbbdb85cfa85412c9a44f1c3a1a4
